### PR TITLE
Cake ingredient names

### PR DESCRIPTION
**Pull request by kroenlein**
Added names to cake.py example so that Graph story has a compliant object

### CODE REVIEW

File: taurus/demo/cake.py [hunk 1]
```diff
@@ -126,27 +136,38 @@ def make_cake():
...
+                                  material=run.material.spec, process=drymix.process.spec)
 
     set_uuids(cake)
 
-    IngredientSpec(material=LinkByUID.from_entity(butter.spec), process=frosting.spec.process)
-    IngredientSpec(material=chocolate.spec, process=frosting.spec.process)
-    IngredientSpec(material=powder_sugar.spec, process=frosting.spec.process)
-    IngredientSpec(material=LinkByUID.from_entity(vanilla.spec), process=frosting.spec.process)
-    IngredientSpec(material=LinkByUID.from_entity(milk.spec), process=frosting.spec.process)
+    print(butter)
```
Comment (bfolie) [inline taurus/demo/cake.py]: debugging statement?

File: taurus/demo/cake.py [hunk 2]
```diff
@@ -126,27 +136,38 @@ def make_cake():
...
+
+    for run in frosting.process.ingredients:
```
Comment (bfolie) [inline taurus/demo/cake.py]: Suggest changing `run` to `ingredient_run`, for readability

File: taurus/demo/cake.py (hunk 1 above)
Comment (kroenlein) [reply taurus/demo/cake.py]: Yes, good catch.

File: taurus/demo/cake.py (hunk 2 above)
Comment (kroenlein) [reply taurus/demo/cake.py]: Changing to ing_run as compromise, given flake8 line length constraint.

**Pull request merged**